README.md

```diff
@@ -558,6 +559,23 @@ import {requestVibration} from '@tef-novum/webview-bridge';
 requestVibration('error');
 ```
 
+### matchContactData
+
+Returns contacts info given an array of phone numbers.
+
+```javascript
+matchContactData: (numbers: Array<string>) => Promise<{
+    matches: {
```

Comment:
instead of `matches` I'd call this `contacts`

Reply:
This structure goes in a NPI team TS, please if you don't have access to this TS, please talk to me and I'll share with you. I prefer to not copy a TS in a public repo

Reply:
having a key makes sense when you want to pick items by key, but in this case the key is unknown, so useless.

If this method returns a collection, use a collection (array of contacts)

Reply:
That makes sense. Let's remove the extra nesting.

Reply:
Great, thanks @jmpg93